tests/functional/api/bulk/annotation_test.py

```diff
@@ -25,13 +25,16 @@ def test_it_accepts_a_valid_request(self, make_request, factories):
             authority_provided_id="1234567890",
             members=[user],
         )
-        factories.AnnotationSlim(
+        annotation_slim = factories.AnnotationSlim(
             user=user,
             group=group,
             shared=True,
             deleted=False,
             created="2018-11-13T20:20:39",
         )
+        factories.AnnotationMetadata(
+            annotation_slim=annotation_slim, data={"some": "value"}
+        )
```

Comment:
Ok, so in this functest we're just adding an `AnnotationMetadata` for the test annotation and asserting that the bulk annotation API returns the expected `metadata`.